pkg/pillar/cmd/zedagent/handlenetworkinstance.go

```diff
@@ -761,6 +761,56 @@ func handleAppFlowMonitorDelete(ctxArg interface{}, key string,
 	log.Infof("handleAppFlowMonitorDelete(%s) done\n", key)
 }
 
+func handleAppVifIPTrigModify(ctxArg interface{}, key string,
+	statusArg interface{}) {
+
+	log.Infof("handleAppVifIPTrigModify(%s)\n", key)
+	ctx := ctxArg.(*zedagentContext)
+	trig := cast.CastVifIPTrig(statusArg)
+	if trig.Key() != key {
+		log.Errorf("handleAppVifIPTrigModify key mismatch %s vs %s; ignored\n",
+			key, trig.Key())
+		return
+	}
+	findVifAndTrigAppInfoUpload(ctx, trig.Key())
+}
+
+func handleAppVifIPTrigDelete(ctxArg interface{}, key string,
+	statusArg interface{}) {
+
+	log.Infof("handleAppVifIPTrigDelete(%s)\n", key)
+	trig := cast.CastVifIPTrig(statusArg)
+	if trig.Key() != key {
```

Comment:
don;t need this if check.. Is Anything - it should be log.Fatalf

Reply:
ok.